debugProtocol.json

```diff
@@ -457,6 +457,10 @@
 										"Debugger attached to an existing process.",
 										"A project launcher component has launched a new process in a suspended state and then asked the debugger to attach."
 									]
+								},
+								"pointerLength": {
+									"type": "number",
+									"description": "The size of a pointer or address for this process, in bits."
```

Comment:
Should we add something like "This can be used by a client to format addresses for display."